examples/hello_from_namespace/hello_from_namespace.cc

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <google/cloud/functions/internal/framework.h>
+
+namespace hello_from_namespace::nested {
```

Comment:
Q: Is the extra `nested` part important?

Reply:
Good question. I think it is important to test nested namespaces (mostly because of how things are implemented). I added a separate example and simplified this one.